src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -579,6 +580,18 @@ public static void enableMilkFluid() {
         enableMilkFluid = true;
     }
 
+    /**
+     * Run this method during mod constructor to enable {@link net.minecraft.FileUtil#RESERVED_WINDOWS_FILENAMES_NEOFORGE} regex being used for filepath validation.
+     * Fixes MC-268617 at cost of vanilla incompat edge cases with files generated with this activated and them migrated to vanilla instance - See PR #767
+     */
+    public static void enableMc268617Fix() {
```

Comment:
I'd prefer a more discoverable name like `enableLenientFilenameValidation`.

Reply:
Well, it's not more lenient since the fixed regex covers cases the original didn't cover too. I'll do enableProperFilenameValidation